test/ansible-memcached/secret/tasks/main.yml

```diff
@@ -12,3 +12,9 @@
           port: 8332
           targetPort: 8332
           name: rpc
+
+- name: Check if jmespath is installed
+  set_fact:
```

Comment:
Not a blocker, especially since this is just a test, but best practice to check for installed Python packages is to use the [python_requirements_info](https://docs.ansible.com/ansible/latest/modules/python_requirements_info_module.html) module